### PR TITLE
lightning: delete content about tpcc

### PR DESCRIPTION
**Pull request by hfxsd**
### What is changed, added or deleted? (Required)

Deleted content about TPCC, and add a limitation note.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v7.4 (TiDB 7.4 versions)
- [x] v7.3 (TiDB 7.3 versions)
- [x] v7.2 (TiDB 7.2 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v7.0 (TiDB 7.0 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):https://github.com/pingcap/docs-cn/pull/14796

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by lance6716 — APPROVED** [review]

File: tidb-lightning/tidb-lightning-physical-import-mode.md [hunk 1]
```diff
@@ -73,13 +73,15 @@ backend = "local"
 
 - 请勿直接使用物理导入模式向已经投入生产的 TiDB 集群导入数据，这将对在线业务产生严重影响。如需向生产集群导入数据，请参考[导入时限制调度范围从集群降低到表级别](/tidb-lightning/tidb-lightning-physical-import-mode-usage.md#导入时暂停-pd-调度的范围)。
 
+- 如果你的 TiDB 生产集群上有延迟敏感型业务，并且并发较小，**不建议**使用 TiDB Lightning 物理导入模式导入数据到该集群，因为可能会影响在线业务。
```
Comment (Frank945946) [inline tidb-lightning/tidb-lightning-physical-import-mode.md:76]:
```suggestion
- 如果你的 TiDB 生产集群上有延迟敏感型业务，并且并发较小，**不建议**使用 TiDB Lightning 物理导入模式（local backend）导入数据到该集群，因为可能会影响在线业务。
```

**Review by Frank945946 — APPROVED** [review]

File: tidb-lightning/tidb-lightning-physical-import-mode.md (hunk 1 above)
Comment (hfxsd) [reply tidb-lightning/tidb-lightning-physical-import-mode.md:76]: 这里不需要加 （local backend），文档开始已经介绍过了

File: tidb-lightning/tidb-lightning-physical-import-mode-usage.md [hunk 2]
```diff
@@ -199,22 +201,6 @@ distsql-scan-concurrency = 3
 switch-mode = '0'
```
Comment (ran-huang) [inline tidb-lightning/tidb-lightning-physical-import-mode-usage.md:201]: 英文里这三行删掉了

**Review by ran-huang — APPROVED** [review]

File: tidb-lightning/tidb-lightning-physical-import-mode-usage.md (hunk 2 above)
Comment (hfxsd) [reply tidb-lightning/tidb-lightning-physical-import-mode-usage.md]: OK 删掉了

**Pull request merged**